packages/react/emotion-utils/src/utils/position.test.tsx

```diff
@@ -46,4 +46,96 @@ describe('position 테스트', () => {
     expect(el).not.toHaveStyleRule('right', '0');
     expect(el).toHaveStyleRule('left', '0');
   });
+
+  describe('position 메서드', () => {
```

Comment:
We are planning to migrate all our tests code to English. What about using English here?

```suggestion
  describe('position methods', () => {
```